dotnet/src/SemanticKernel.Core/PromptTemplate/NoopPromptTemplateFactory.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Provides an implementation of <see cref="IPromptTemplateFactory"/> which creates no operation instances of <see cref="IPromptTemplate"/>.
+/// </summary>
+public sealed class NoopPromptTemplateFactory : IPromptTemplateFactory
```

Comment:
Alternate names instead of Noop could be Echo, Passthrough, Null,...

Reply:
Indeed, a NOOP I would expect no template as return, always empty, or `Null`